### ISSUE TITLE
Add SR-only table to Today

### ISSUE THREAD

**Issue by partly-igor** [weather-gov/weather.gov#1988]
### Description
We have a chart on the top of the today tab displaying the temperatures for the day. We need to add a hidden table with these data points alongside the chart so that there is an accessible alternative to the chart. 

### Acceptance Criteria
- [ ] Table of temperatures and feels-like is added to the today tab alongside the chart
- [ ] Table is SR-only / not visible. 
- [ ] Chart doesn't announce "empty group" for the chart to screenreaders

` Tag the related milestone to this story`
